Formula/k9s.rb

```diff
@@ -25,6 +25,7 @@ def install
       -X github.com/derailed/k9s/cmd.version=#{version}
       -X github.com/derailed/k9s/cmd.commit=#{Utils.git_head}
     ]
+    ENV["CGO_ENABLED"] = "0"
```

Comment:
Could you please add a comment above this line (starts with a `#`) explaining why `k9s` must be built with cgo disabled? Having the reason here would help to ensure that this setting isn't inadvertently removed in the future.